pkgs/by-name/up/uppaal/package.nix

```
@@ -0,0 +1,80 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  makeWrapper,
+  jdk17,
+  makeDesktopItem,
+  copyDesktopItems,
+  wrapGAppsHook,
+  shared-mime-info,
+  gdk-pixbuf,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "uppaal";
+  version = "5.0";
+  subversion = "0";
+  platform = "linux64";
+
+  src = fetchzip {
+    url = "https://download.uppaal.org/uppaal-${version}/uppaal-${version}.${subversion}/uppaal-${version}.${subversion}-${platform}.zip";
+    hash = "sha256-o71mP2/sDNRpmA1Qx59cvx6t4pk5pP0lrn1CogN3PuM=";
+  };
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "uppaal";
+      exec = "uppaal %U";
+      icon = "uppaal";
+      comment = "real-time modelling and verification tool";
+      desktopName = "Uppaal";
+      genericName = "Uppaal";
+      categories = [ "Development" ];
+    })
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+    wrapGAppsHook
+  ];
```

`copyDesktopItems` still needs to go here to have the intended effect

```suggestion
  nativeBuildInputs = [
    copyDesktopItems
    makeWrapper
    wrapGAppsHook
  ];
```